src/relay/op/tensor/reduce.cc

```diff
@@ -266,7 +269,7 @@ bool ArgReduceRel(const Array<Type>& types, int num_inputs, const Attrs& attrs,
   CHECK(param != nullptr);
 
   // assign output type and shape
-  auto oshape = ReduceShapeImpl(in_shape, param, reporter);
+  auto oshape = ReduceShapeImpl<ReduceAttrs>(in_shape, param, reporter);
```

Comment:
no need to add explicit template annotation, it should be inferred

Reply:
They have been fixed. Thanks.